lms/djangoapps/grades/tasks.py

```diff
@@ -24,9 +26,27 @@
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
 def recalculate_subsection_grade(
+        # pylint: disable=unused-argument
         user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
 ):
     """
+    Shim to allow us to modify this task's signature without blowing up production on deployment.
```

Comment:
I think we should be fine - previously queued tasks will have an arguments list of `user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs`. From that, we drop `weighted_earned` and `weighted_possible`, and shim in a dummy value for `task_queued_time`, which allows us to call the new method signature cleanly.